### PR TITLE
Migrate to Camunda Modeler 5

### PR DESCRIPTION
**Pull request by barmac**
This PR migrates the plugin to Camunda Modeler 5 APIs together with the new properties panel.

Before it can be merged, `camunda-modeler-plugin-helpers` dependency must be updated when [this PR](https://github.com/camunda/camunda-modeler-plugin-helpers/pull/16) is merged and released. The bundle will need to be re-generated.

### CODE REVIEW

**Comment by barmac** [comment]
Check out the new look at https://github.com/camunda/camunda-modeler-process-io-specification-plugin/tree/migrate-to-properties-panel-1-0-0

**Comment by barmac** [comment]
This is ready to be merged.

File: client/properties-panel/process-io-extension-provider.js [hunk 1]
```diff
@@ -90,36 +16,38 @@ function createProcessIoTab(element, injector) {
...
 ProcessIoExtensionProvider.$inject = [
+  'propertiesPanel',
   'injector'
-];
\ No newline at end of file
+];
+
+
+function getProcessRef(element) {
+  var bo = getBusinessObject(element);
```
Comment (pinussilvestrus) [inline client/properties-panel/process-io-extension-provider.js]:
```suggestion
  const bo = getBusinessObject(element);
```

**Review by pinussilvestrus — COMMENTED** [review]
Works nice! 👏 

Should we maybe add a [compatibility notice](https://github.com/pinussilvestrus/camunda-modeler-excel-import-plugin#compatibility-notice) to this plugin to explain which version of the Modeler is compatible?

**Comment by barmac** [comment]
Great idea! I added this via https://github.com/camunda/camunda-modeler-process-io-specification-plugin/pull/9/commits/eb44c361d32d9586be21162084776ebbd19bf10b

**Comment by barmac** [comment]
I've changed the link and added CHANGELOG.

**Review by nikku — COMMENTED** [review]
Thanks! I can confirm it works for me.

Just found one, rather surprising bit of interaction: 

capture yac91T_optimized

**Comment by barmac** [comment]
> Just found one, rather surprising bit of interaction:

Do you mean that the name in extension elements is focused automatically or something else?

**Comment by nikku** [comment]
I mean two things:

* the extension element section opens automagically => actual editing does happen elsewhere
* the last, rather than the first section that "automatically opened" is focussed => I'd assume the first one to be focussed.

**Comment by barmac** [comment]
Makes sense to me. Not sure if it's easily fixable, but I will check that. Thanks for reporting.

**Pull request merged**